src/components/hacks/aurora/stats.tsx

```diff
@@ -1,7 +1,28 @@
+"use client";
 import Image from "next/image";
 import auroraStar from "@/public/logos/aurorastar.webp";
+import { useQuery } from "@tanstack/react-query";
 
 const Stats = () => {
+  const fetchClosedPRs = async () => {
+    const res = await fetch(
+      "https://api.github.com/search/issues?q=repo:acm-ucr/aurora+type:pr+state:closed",
+    );
+
+    const data = await res.json();
+    return data.total_count;
+  };
+
+  const {
+    data: closedPRs,
+    isLoading,
+    isError,
+  } = useQuery({
+    queryKey: ["closedPRs"],
+    queryFn: fetchClosedPRs,
+    staleTime: 1000 * 60 * 5,
+  });
+
   return (
     <div className="py-30 flex flex-col justify-center">
       <div className="pb-18 ml-[15%] grid gap-10 text-acm-gray-500 [grid-template-columns:23%_21%_23%]">
```

Comment:
leave a little less space on the right of the number of PRs
```suggestion
      <div className="pb-18 ml-[15%] grid gap-10 text-acm-gray-500 [grid-template-columns:23%_16%_23%]">
```